moveit_setup_assistant/CMakeLists.txt

```diff
@@ -1,169 +1,142 @@
-  src/widgets/perception_widget.cpp
-  src/widgets/configuration_files_widget.cpp
-  src/widgets/ros_controllers_widget.cpp
-  src/widgets/controller_edit_widget.cpp
-  src/widgets/navigation_widget.cpp
-  src/widgets/header_widget.cpp
-  src/widgets/setup_assistant_widget.cpp
-  src/widgets/setup_screen_widget.cpp
-  src/widgets/simulation_widget.cpp
-  src/widgets/author_information_widget.cpp
-  ${HEADERS}
+target_include_directories(${PROJECT_NAME}_tools PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include>
+  ${OMPL_INCLUDE_DIRS}
 )
-set_target_properties(${PROJECT_NAME}_widgets PROPERTIES VERSION "${${PROJECT_NAME}_VERSION}")
-target_link_libraries(${PROJECT_NAME}_widgets
+ament_target_dependencies(
   ${PROJECT_NAME}_tools
-  ${QT_LIBRARIES}
-  ${catkin_LIBRARIES}
-  ${Boost_LIBRARIES}
+  moveit_core
+  moveit_ros_planning
+  moveit_ros_visualization
+  rclcpp
+  urdf
+  srdfdom
 )
 
-# executables
-add_executable(${PROJECT_NAME} src/setup_assistant_main.cpp)
-target_link_libraries(${PROJECT_NAME}
-  ${PROJECT_NAME}_widgets ${PROJECT_NAME}_tools
-  ${QT_LIBRARIES} ${catkin_LIBRARIES} ${Boost_LIBRARIES} log4cxx
+add_executable(${PROJECT_NAME}_updater src/collisions_updater.cpp)
+target_include_directories(${PROJECT_NAME}_updater PUBLIC
+  $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+  $<INSTALL_INTERFACE:include>)
+target_link_libraries(
+  ${PROJECT_NAME}_updater
+  ${PROJECT_NAME}_tools
 )
 
-add_executable(${PROJECT_NAME}_updater src/collisions_updater.cpp)
-target_link_libraries(${PROJECT_NAME}_updater
-  ${PROJECT_NAME}_tools ${catkin_LIBRARIES} ${Boost_LIBRARIES})
-set_target_properties(${PROJECT_NAME}_updater
-                      PROPERTIES OUTPUT_NAME collisions_updater
-                      PREFIX "")
+ament_target_dependencies(
+  ${PROJECT_NAME}_updater
+  moveit_core
+  moveit_ros_planning
+  moveit_ros_visualization
+  rclcpp
+  urdf
+  srdfdom
+)
 
-install(TARGETS ${PROJECT_NAME} ${PROJECT_NAME}_updater
-  RUNTIME DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
+set_target_properties(
+  ${PROJECT_NAME}_updater
+  PROPERTIES OUTPUT_NAME collisions_updater
+  PREFIX ""
+)
 
-install(TARGETS ${PROJECT_NAME}_widgets ${PROJECT_NAME}_tools
-  LIBRARY DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  ARCHIVE DESTINATION ${CATKIN_PACKAGE_LIB_DESTINATION}
-  RUNTIME DESTINATION ${CATKIN_GLOBAL_BIN_DESTINATION})
+install(
+  TARGETS
+    ${PROJECT_NAME}_tools
+    ${PROJECT_NAME}_updater
+  EXPORT ${PROJECT_NAME}Targets
+  ARCHIVE DESTINATION lib
+  LIBRARY DESTINATION lib
+  RUNTIME DESTINATION lib/${PROJECT_NAME}
+  INCLUDES DESTINATION include
+)
+install(DIRECTORY launch DESTINATION share/${PROJECT_NAME})
+install(DIRECTORY resources DESTINATION share/${PROJECT_NAME})
+install(DIRECTORY templates DESTINATION share/${PROJECT_NAME})
+ament_export_targets(${PROJECT_NAME}Targets HAS_LIBRARY_TARGET)
+ament_export_dependencies(${THIS_PACKAGE_INCLUDE_DEPENDS})
```

Comment:
Did you define this somewhere?  I don't see it.  Look at the one in moveit_core for an example.

Reply:
This was a stright up copy/paste from your previous comment.

Reply:
Add the list